src/components/asset-item.tsx

```diff
@@ -56,14 +55,16 @@ export const AssetItem = memo(
             <Stack flexGrow={1}>
               <SpaceBetween width="100%">
                 <Text>{title}</Text>
-                <Tooltip
-                  placement="left-start"
-                  label={formatted.isAbbreviated ? amount : undefined}
-                >
-                  <Text fontVariantNumeric="tabular-nums" textAlign="right">
-                    {formatted.value}
-                  </Text>
-                </Tooltip>
+                <Box>
```

Comment:
See my comment: https://github.com/blockstack/stacks-wallet-web/pull/1291#issuecomment-860734537

Reply:
I'm not entirely sure what's being suggested here.

We want to keep things predictable. `Tooltip` inherently passing props down an extra level to its children is a behaviour that requires knowledge of the inner implementation (something that should be done sparingly). Plus, what if there are multiple children?

It's not clear to me why this is wrong?
```html
  <Box>
    <Tooltip label="hi kyran">
      some component that now has mb
    </Tooltip>
  </Box>
```

I'm sold on the `<Stack>` component as a helpful spacing tool. But as it is, our implementation is confusing. I don't see why its children **must** support spacing props, versus it wrapping them/handling spacing another way. It silently fails when a child doesn't pass down style props (an invisible rule). [Material](https://next.material-ui.com/components/stack/) and [Chakra](https://chakra-ui.com/docs/layout/stack) both work with any child, and don't have implicit requirements like ours. These are the pain points of implementing/maintaining our own UI library 🤷🏼‍♂️  

In my personally hierarchy of important, runtime errors trumps an extra wrapping <Box> element. Can we do this in another PR? 🙏🏼 

this tooltip reimplementation, so can we do this in another PR?

Reply:
Sure, let's keep what you have here for now, and I'll follow up on your comments (and try to explain why `Stack` is amazing ;) ) in a new PR in the UI library where these changes should go.